src/bestcrypt_ve_fmt_plug.c

```diff
@@ -0,0 +1,456 @@
+/*
+ * JtR format to crack password protected BestCrypt v4 volume
+ * This implementation relies on findings from @trounce1 and @kholia 
+ * reverse-engineering work
+
+ * ----------------------------------------------------------------------------
+ * "THE BEER-WARE LICENSE" (Revision 42):
+ * <jean-christophe.delaunay@synacktiv.com> wrote this file. As long as you 
+ * retain this notice you can do whatever you want with this stuff. If we meet 
+ * some day, and you think this stuff is worth it, you can buy me a beer in 
+ * return
+ * ----------------------------------------------------------------------------
+ */
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_bestcrypt_ve;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_bestcrypt_ve);
+#else
+
+#include <string.h>
+#include <errno.h>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#include "arch.h"
+#include "misc.h"
+#include "common.h"
+#include "formats.h"
+#include "params.h"
+#include "options.h"
+#include "yescrypt/yescrypt.h"
+#include "aes.h"
+#include "twofish.h"
+#include "serpent.h"
+#include <openssl/camellia.h>
+#include "sha.h"
+#include "jumbo.h"
+
+#define FORMAT_NAME             "BestCrypt Volume Encryption v4 "
+#define FORMAT_LABEL            "Jetico BestCrypt"
```

Comment:
Must not have a space inside the label. It should be e.g. just `BestCryptVE`, or maybe `BestCrypt4` if we don't intend to add other versions to this same format.